core/templates/services/site-analytics.service.ts

```diff
@@ -137,6 +137,30 @@ export class SiteAnalyticsService {
     );
   }
 
+  registerClickCreateAccountButtonEvent(): void {
+    this._sendEventToLegacyGoogleAnalytics(
```

Comment:
@AkashPaloju I don't think legacy GA works any more, we should remove it. Let's use the non-legacy version of this instead.

Also could you please add another parameter along with the events common to both pages that allows us to pass along the value of the feature flag? Then we can roll this out at 50% and see how the click rates compare between the two.